package.json

```diff
@@ -15,14 +15,15 @@
     "types/*.d.ts"
   ],
   "scripts": {
-    "build": "rollup -c && cp dist/chartjs-chart-financial.js docs/",
+    "build": "rollup -c && copyfiles dist/chartjs-chart-financial.js docs/",
     "package": "npm run build && node scripts/create-packages.js",
     "lint": "eslint docs/**/*.html src/**/*.js test/**/*.js",
     "lint-types": "eslint \"types/**/*.ts\" && tsc -p types/tests/",
     "test": "npm run test-types && npm run test-unit",
     "test-unit": "karma start --single-run --coverage --grep",
     "test-unit:dev": "karma start --auto-watch --grep",
-    "test-types": "tsc -p types/tests/"
+    "test-types": "tsc -p types/tests/",
+    "test-node": "node test/node.js"
```

Comment:
why have a separate entry point for this? can we just make this a normal test in `test/specs` like the others?

Reply:
The test needed to run on Node not via karma on the browser and this was the simplest way I could think of doing it

Reply:
Also, not sure why CI is failing at finding file `./coverage/chrome/lcov.info`, is this working in master? I did change some code in that space but a brief review of the changes seem to indicate that it should not change this setting.